Libraries/LibWeb/HTML/StructuredSerialize.cpp

```diff
@@ -1047,44 +1063,82 @@ class Deserializer {
     GC::RootVector<JS::Value> m_memory; // Index -> JS value
     size_t m_position { 0 };
 
-    static GC::Ref<Bindings::PlatformObject> create_serialized_type(StringView interface_name, JS::Realm& realm)
+    static bool is_serializable_interface_exposed_on_target_realm(SerializeType name, JS::Realm& realm)
```

Comment:
Just an idea for the future, but I wonder if we can generate `is_serializable_interface_exposed_on_target_realm` and `create_serialized_type`? Would be nice to not have to maintain these large switches